Added forms

ndought opened the pull request:
Added hashtags and reviews forms.

ndought committed:
Added forms

Added hashtags and reviews forms.

AbdulJ23 committed:
Added forms

Added hashtags and reviews forms.

AbdulJ23 committed:
Merge remote-tracking branch 'origin/html-forms' into html-forms

ColinWake commented on src/test/java/org/wecancodeit/reviews/controllers/ReviewControllerTest.java:
```diff
@@ -14,7 +14,9 @@
-public class ReviewControllerTest {
+public class
+
+ReviewControllerTest {
```
Please line this up

ColinWake replied:
Fixed in https://github.com/2020-Spring-Cohort/reviews-site-models/commit/4a8682409c75e91bd225174719d1faef4904f7a4

fpemba commented on src/test/java/org/wecancodeit/reviews/controllers/ReviewControllerTest.java:
```diff
@@ -14,7 +14,9 @@
-public class ReviewControllerTest {
+public class
+
+ReviewControllerTest {
```
Please combine these three lines.

ColinWake commented on src/main/resources/templates/hashtags.html:
```diff
@@ -15,6 +15,10 @@ <h2 th:each="hashtag : ${hashtags}">
+<form action="/hashtags/add-reviews" method="POST">
+    <input type="text" name="title" placeholder="Hashtag name"/>
+    <button type="submit">Add Hashtag</button>
+</form>
```
Make sure `name` attribute corresponds to the `@RequestParam` in the post mapping

ndought replied:
fixed via https://github.com/2020-Spring-Cohort/reviews-site-models/pull/41/commits/4a8682409c75e91bd225174719d1faef4904f7a4#diff-fd2eeff5cc477fbeadd8f39a1f01e363

ColinWake commented on src/main/resources/templates/reviews.html:
```diff
@@ -17,5 +17,16 @@ <h3 th:text="${review.price}" class="review__price">Default Price</h3>
 
 </div>
 
+<form action="/reviews/add-reviews" method="POST">
+    <input type ="text" name="name" placeholder="Review Name" />
+    <button type="submit">Add Review</button>
+</form>
```
Review takes more args than this

ndought replied:
fixed via https://github.com/2020-Spring-Cohort/reviews-site-models/pull/41/commits/4a8682409c75e91bd225174719d1faef4904f7a4#

fpemba commented on src/main/resources/templates/reviews.html:
```diff
@@ -17,5 +17,16 @@ <h3 th:text="${review.price}" class="review__price">Default Price</h3>
 </div>
 
+<form action="/reviews/add-reviews" method="POST">
+    <input type ="text" name="name" placeholder="Review Name" />
```
Instead of review name, should just be "review"

ndought committed:
small edit to form and test

Lined up ReviewControllerTest as requested in pull request. Also fixed hashtags.html form as requested in pull request.

ndought committed:
Merge remote-tracking branch 'origin/html-forms' into html-forms

ColinWake reviewed (approved):
Nice!
